includes/controllers/pages/class-manage-listings.php

```diff
@@ -82,6 +82,11 @@ public function maybe_remove_listing_buttons( $res, $listing_id, $user_id ) {
 
     }
 
+    /**
+     * Show different payment url for a pending listing
+     *
+     * @since x.x
```

Comment:
Maybe we should remove this since because it won't be accurate?

Reply:
Updated the comment and removed the `since` tag as there is no reference on the changes or when the function was added.